src/pages/en/guides/data-fetching.md

```diff
@@ -4,35 +4,109 @@ title: Data Fetching
 description: Learn how to fetch remote data with Astro using the fetch API.
 ---
 
-Astro components and pages can fetch remote data to help generate your pages. Astro provides two different tools to pages to help you do this: **fetch()** and **top-level await.**
+Astro pages can fetch remote data at build time to help generate your pages.
 
-## `fetch()`
+## `fetch()` in Astro
 
-Astro pages have access to the global `fetch()` function in their setup script. `fetch()` is a native JavaScript API ([MDN<span class="sr-only">- fetch</span>](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch)) that lets you make HTTP requests for things like APIs and resources.
+All [Astro components](/en/core-concepts/astro-components) have access to the global `fetch()` function in their component script to make HTTP requests to APIs. This fetch call will be executed at build time, and the data will be available to the component template for generating dynamic HTML. 
```

Comment:
Given the link to top-level `await` docs, might it also be helpful to link to docs for `fetch` here? For example the equivalent MDN page: https://developer.mozilla.org/en-US/docs/Web/API/fetch

Not sure how best to judge the likelihood that a dev will be familiar with the Fetch API already.

Reply:
Yeah, I'm undecided as to whether fetch really needs an MDN reference. But, I guess, if it was in there before, and if we do have it for top-level await (which I think is a newer concept, and why I kept it)... all it really does is make the word orange. No big sacrifice for us!